### DISCUSSION TITLE
Feature Request: IPv6 support

### DISCUSSION THREAD

**Discussion by dgcampea** [jcorporation/myMPD#701] [Ideas]
According to https://github.com/jcorporation/myMPD/issues/476, mympd seems to be IPv4 only, can support for IPv6 be added for compatibility with native IPv6-only networks?

**Comment by jcorporation** [comment]
myMPD is compiled with IPv6 support, but it is untested. There is no support for acls for IPv6, but the rest could work. Give it a try and I can fix issues poping up.

**Reply by dgcampea** [reply]
Tried setting `http_host` to `[::]` and `::`, both fail but in different ways:
- `[::]` is invalid and crashes.
- `::` seems valid (doesn't crash and mympd says `listening to http://:::<port>`) but neither `localhost`, `localhost6` nor `localhost4` (or their IP equivalents) work.

**Comment by jcorporation** [comment]
Sorry, I forgot myMPD defaults to disabled IPv6: https://jcorporation.github.io/myMPD/installation/compiling

I tested it and with [::] myMPD listens on ipv6.

**Comment by jcorporation** [comment]
IPv6 is now enabled by default in devel.
